pytravharv/SubjPropPathAssertion.py

```diff
@@ -4,7 +4,7 @@
 from pyrdfj2 import J2RDFSyntaxBuilder
 from pytravharv.TargetStore import TargetStore
 from pytravharv.TravHarvConfigBuilder import AssertPath
-from pytravharv.WebAccess import WebAccess
+from pytravharv.WebAccess import web_access
```

Comment:
Also, as a function name `web_access` doesn't convey a lot of information. What does it access on the web?